Project-iOS/XLProjectName/XLProjectName/Helpers/Networking/PaginationViewModel.swift

```diff
@@ -15,9 +15,14 @@ class PaginationViewModel<Element: Decodable where Element.DecodedType == Elemen
     
     var request: PaginationRequest<Element>
     
+    let forceRefreshTrigger = PublishSubject<Void>()
```

Comment:
Have you thought about using only one refreshTrigger of type PublishSubject<bool>()

In this case emitted items should represent if the refresh should cancel pending request or not (force or not).